src/main/java/seedu/address/logic/commands/FindCommand.java

```diff
@@ -25,21 +25,26 @@ public class FindCommand extends Command {
 
     public static final String COMMAND_WORD = "find";
 
+    public static final String INVALID_FIND_ARGS_MESSAGE = "Arguments of the Find command should not contain '/' as it "
+            + "is reserved to be used with a prefix to define the attribute to find by. "
+            + "Please ensure that the prefix used is valid.";
+
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Except for Salary attribute, finds all persons whose "
             + "attributes contain the keyword (case-insensitive) specified for those attributes and displays them as a "
-            + "list with index numbers. For Salary, finds all persons whose salary is within a range instead.\n"
+            + "list with index numbers. For Salary, finds all persons whose salary is within a range instead. "
+            + "Upper bound of the range should be less than the maximum allowed salary which is 1 000 000 000.\n"
```

Comment:
You can consider using `Salary.MAXIMUM_SALARY` here, so that any changes made to `Salary.MAXIMUM_SALARY` will be reflected here automatically.

Reply:
Ok!

Reply:
Ok! I have made the change